packages/fxa-content-server/tests/functional/lib/helpers.js

```diff
@@ -2072,7 +2098,8 @@ const testElementWasShown = thenify(function (selector, message) {
         // remove the attribute so subsequent checks can be made
         // against the same element. displaySuccess and displayError
         // will re-add the 'data-shown' attribute.
-        $(selector).removeAttr('data-shown');
+        // TODO experimentally disabling this.
+        //$(selector).removeAttr('data-shown');
```

Comment:
Good catch. Probably the best thing to do would be to switch uses of `testElementWasShown` to `testElementExists` for the new-settings tests, since the `data-shown` attribute is still useful in testing the old backbone code. I'll re-enable the `data-shown` check and update the tests.

Reply:
Welp, making these changes has gotten complicated. I've filed #7882 as a followup.